src/status_im2/subs/root.cljs

```diff
@@ -195,6 +195,10 @@
 (reg-root-key-sub :wallet/swap-to-token-amount :wallet/swap-to-token-amount)
 (reg-root-key-sub :wallet/swap-advanced-mode? :wallet/swap-advanced-mode?)
 
+;;; Wallet 2
+(reg-root-key-sub :wallet-2/tokens :wallet-2/tokens)
```

Comment:
A better versioning approach in my opinion is to rename the old events and keep the new names as good as we want, without polluting them with versioning. So the opposite of what we're doing in this case. To me, the new code in `status_im2/` should be 100% of what we want and the old code should be modified to look as ugly as need them to be.

Why the version though? The event is already used inside `status-im2` and the keyword `:wallet/tokens` is not used.

Reply:
Im okay adjusting the approach taken here if there is some strong consensus. Albeit I think this should be done outside of this pr.

With that said, I prefer the current approach in this pr as it is more consistent with how the migration has been done so far. Everything in the new ui is marked with ‘-2’ to some degree so far, resources, namespaces etc and also for onboarding we took this approach. 

In the end, once all of the old code is deleted it will be a simple search and replace to clean up these events and subs.

The versioning here is added so that when we go to remove these old wallet events and subs (regardless of whether they have a matching old/new version ) there is no doubt to what is old code and we can have confidence in what to remove.

Reply:
@cammellos any thoughts on this :)

Reply:
> Albeit I think this should be done outside of this pr.

For sure, we should never mix large renames with feature PRs.

> With that said, I prefer the current approach

It's nothing serious, but I shared my perspective here because it's a smell that the new code has to use versioning in itself. But I get it, that's only because of the fact that keywords are global names.

In the end, I think this versioning scheme creates more dev effort because we have to continuously worry about name clashes with the old code, and when we delete the old code we need to update the new one.

A simpler approach is to rename the old wallet events and subscriptions to `old-<whatever`, and then the new code won't ever conflict and it won't require additional refactoring to rename. This same strategy could be used in other parts of the code.

Reply:
Btw wrt "For sure, we should never mix large renames with feature PRs." - 

I was not referring to the work to change the events name etc, I meant that this discussion would be better to have outside of this pr.

Reply:
> I was not referring to the work to change the events name etc, I meant that this discussion would be better to have outside of this pr.

Agreed, my original point was just to start something going.

Reply:
Yep ofc! 👌

Reply:
I don't have a strong opinion myself, whatever minimizes potential bugs from refactoring :)